app/jobs/calendar_importer/parsers/zarts.rb

```diff
@@ -4,6 +4,8 @@
 
 module CalendarImporter::Parsers
   class Zarts < Xml
```

Comment:
Lets use this opportunity to convert this to a proper ticketsolve importer (i.e. adjust whitelist url, rename etc)

Reply:
so rename Zarts -> TicketSolve (change filename), change the `NAME` and the `DOMAIN = %w[ticketsolve.com]` and the pattern should be something like `/^https?:\/\/([^.]*).ticketsolve.com\.*/` ?

Reply:
Yep exactly (no capitalised S tho by the look of it)